main.c

```diff
@@ -10,17 +10,18 @@
 #include <uart_cbus.h>
 #include <rtc.h>
 #include <pci.h>
+#include "memory_allocator.h"
 
 typedef struct cpuinfo {
```

Comment:
This change does not relate to what you're implementing. If so please remove irrelevant modification and postpone it to another patch.